README.md

```diff
@@ -40,12 +40,12 @@ git clone https://github.com/morinlab/gambl
 git clone https://github.com/morinlab/GAMBLR
 ```
 
-3. Update the **paths** in your local [config.yml](https://github.com/morinlab/GAMBLR/blob/master/config.yml) to point to the recently cloned, local **gambl** folder (repo_base). In your favorite text editor, edit the line shown below (under *remote*). Similarly, you will also need to edit the line above it to point to where you will eventually sync the GAMBL results.
+3. Update the **paths** in your local [config.yml](https://github.com/morinlab/GAMBLR/blob/master/config.yml) (GAMBLR-master) to point to the recently cloned, local **gambl** folder (repo_base). In your favorite text editor, edit the line shown below (under *remote*). Similarly, you will also need to edit the line above it to point to where you will eventually sync the GAMBL results.
 
 ```
 remote:
     project_base: "/path/to/your/local/gambl_results_directory/"
-    repo_base: "/path/to/your/local/gambl_repo/"
+    repo_base: "~/git_repos/gambl-master/"
```

Comment:
maybe it is better for this to stay as a placeholder since this is not executed anyways and just shows the user where this path should point to?

Reply:
I'm also fine with using placeholders instead. The reason that I updated this was that I wanted it to match what's set up if you follow along with the guide in the README. I'll revert this. Thanks!